src/graphql/outlier.rs

```diff
@@ -121,19 +121,19 @@ impl OutlierMutation {
         &self,
         ctx: &Context<'_>,
         #[graphql(validator(min_items = 1))] input: Vec<PreserveOutliersInput>,
-    ) -> Result<usize> {
+    ) -> Result<Vec<PreserveOutlierOutput>> {
         let store = super::get_store(ctx).await?;
         let map = store.outlier_map();
-        let mut updated = vec![];
+        let mut updated: Vec<PreserveOutlierOutput> = vec![];
         for outlier_key in input {
-            let outlier_id = (outlier_key.id, outlier_key.source.clone());
+            let preserve: PreserveOutliersInput = outlier_key.clone();
             let key: review_database::OutlierInfoKey = outlier_key.into();
             if map.update_is_saved(&key)? {
-                updated.push(outlier_id);
+                updated.push(PreserveOutlierOutput::from(preserve));
```

Comment:
I don't think this logic serves the purpose of the issue, which said

> Instead of returning the count of successfully marked outliers, the endpoint should return a list of outliers that were **not** marked as "saved" due to their absence in the database.

I styled "not" in bold, to make it stand out.

Could you check this please?